### PR TITLE
adding paralus docs to mkdocs file

### PR DESCRIPTION
**Pull request by parkand1**
Adding paralus docs to mkdocs file as it currently does not show up in the navigation panel via public documentation page

### CODE REVIEW

**Review by shapirov103 — COMMENTED** [review]
Let's add them to https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/index.md as well.

File: docs/addons/index.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ The framework currently supports the following add-ons.
 | [`NewRelicAddOn`](./newrelic.md)                                       | Adds [New Relic](https://newrelic.com/) and [Pixie](https://pixielabs.ai/) observability for Amazon EKS.                               | ✅ | 
 | [`NginxAddOn`](./nginx.md)                                             | Adds NGINX ingress controller  | ✅ | ✅ |                                                                                                        |
 | [`OpaGatekeeperAddOn`](./opa-gatekeeper.md)                            | Adds OPA Gatekeeper                                                                                                                    | ✅ | ✅ |
+| [`ParalusAddOn`](./paralus.md)                            | Adds [Paralus](https://paralus.io/)                                                                                                                   | ✅ | ✅ |
```
Comment (shapirov103) [inline docs/addons/index.md:60]: Just making sure you verified it - paralus supports arm64? I see a checkbox set for it.

**Review by shapirov103 — COMMENTED** [review]
Please see my question

File: docs/addons/index.md (hunk 1 above)
Comment (niravparikh05) [reply docs/addons/index.md:60]: yeah, paralus works on arm processors as well

**Review by shapirov103 — APPROVED** [review]
LGTM

**Comment by shapirov103** [comment]
This will be published with the next minor release.

**Pull request merged**